platform/fio/transaction.go

```diff
@@ -1,9 +1,81 @@
 package fio
 
 import (
+	"encoding/json"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+	"strconv"
+	"strings"
+	"time"
 )
 
 func (p *Platform) GetTxsByAddress(address string) (page blockatlas.TxPage, err error) {
-	return page, err
+	// take actor from address
+	account := actorFromPublicKeyOrActor(address)
+	actions, err := p.client.getTransactions(account)
+	if err != nil {
+		return nil, err
+	}
+	var txs []blockatlas.Tx = make([]blockatlas.Tx, 0)
+	for _, a := range actions {
+		tx, err := p.Normalize(&a, account)
+		if err != nil {
+			continue
+		}
+		txs = append(txs, tx)
+	}
+	txPage := blockatlas.TxPage(txs)
+	return txPage, nil
+}
+
+func (p *Platform) Normalize(action *Action, account string) (blockatlas.Tx, error) {
+	var (
+		to, from string
+		amount   blockatlas.Amount
+	)
+
+	// Action Act.Name == "trnsfiopubky" not handled
+	if action.ActionTrace.Act.Account == "fio.token" &&
+		action.ActionTrace.Act.Name == "transfer" {
+		//if action.ActionTrace.Act.Name == "transfer" {
+		// convert to action-specific data
+		var actionData ActionData
+		dataJson, _ := json.Marshal(action.ActionTrace.Act.Data)
+		dataErr := json.Unmarshal(dataJson, &actionData)
+		if dataErr != nil {
+			return blockatlas.Tx{}, errors.E("Unparseable Data")
+		}
+		from = actionData.From
+		to = actionData.To
+		amountNum, err := strconv.ParseFloat(strings.Split(actionData.Quantity, " ")[0], 64)
+		if err == nil {
+			amount = blockatlas.Amount(strconv.Itoa(int(amountNum * 1000000000)))
+		}
+		//}
+		//if action.ActionTrace.Act.Name == "trnsfiopubky" {
```

Comment:
Do we need these comments?

Reply:
I am unsure of the different actions, now only "transfer" is returned, but e.g. trnsfiopubky not (initial transfer to account?).  Anyway, I clean up; if later it has to be extended, it will be added.